### PR TITLE
Invite page

### PR DESCRIPTION
**Pull request by dan-online**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # in the templates via {{ site.myvariable }}.
 title: DuncteBot
 guild_invite: https://dashboard.dunctebot.com/api/joinGuild
-bot_invite: https://dashboard.dunctebot.com/invite
+bot_invite: https://discordapp.com/oauth2/authorize?client_id=210363111729790977&scope=bot&permissions=-1
```
Comment (duncte123) [inline _config.yml]: Please revert this change

File: invite.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Home
+---
+<div class="row flow-text">
```
Comment (duncte123) [inline invite.html:6]: This should use the meta tag, or preferably this plugin https://github.com/jekyll/jekyll-redirect-from

**Review by duncte123 — CHANGES_REQUESTED** [review]

File: invite.html (hunk 2 above)
Comment (dan-online) [reply invite.html:6]: Meta tag is for the head which I can't add so Ill check the plugin

File: _config.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@
 # in the templates via {{ site.myvariable }}.
 title: DuncteBot
 guild_invite: https://dashboard.dunctebot.com/api/joinGuild
-bot_invite: https://dashboard.dunctebot.com/invite
+bot_invite:  https://dashboard.dunctebot.com/invite
```
Comment (duncte123) [inline _config.yml]: Too many spaces here

File: Gemfile [hunk 4]
```diff
@@ -30,3 +30,6 @@ gem "wdm", "~> 0.1.0" if Gem.win_platform?
 
 # Check if the html is valid
 gem "html-proofer", "~> 3.10.2"
+
+# Redirect plugin
+gem 'jekyll-redirect-from'
```
Comment (duncte123) [inline Gemfile:35]: Add the plugins in the plugins group

**Review by duncte123 — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 4 above)
Comment (dan-online) [reply Gemfile:35]: The commented out part?

Comment (duncte123) [reply Gemfile:35]: Yup, it states with a comment above that the plugins should go there

Comment (dan-online) [reply Gemfile:35]: Oh ok lol I thought cause it was commented out that it wouldn't work hahaha

**Review by dan-online — COMMENTED** [review]
Can this be merged?

**Comment by duncte123** [comment]
We stopped using jekyll

**Comment by dan-online** [comment]
> We stopped using jekyll

Oh ok haha
